tritium-core/src/main/java/com/palantir/tritium/event/Handlers.java

```diff
@@ -29,6 +29,13 @@ public final class Handlers {
 
     private static final SafeLogger log = SafeLoggerFactory.get(Handlers.class);
 
+    /**
+     * Returns a disabled {@link InvocationContext}.
+     */
+    public static InvocationContext disabled() {
+        return DisabledHandlerSentinel.INSTANCE;
+    }
```

Comment:
This is reasonable to expose, but we should document that it should only be used to pass directly to other methods in `Handlers`.